.github/CODEOWNERS

```diff
@@ -0,0 +1,39 @@
+# Default owners for everything in the repository:
+* @darkeld3r @aleksuss
+
+# Javascript specific changes:
+* @darkeld3r @boguslavsky
+
+# Cryptography:
+crypto/* @darkeld3r @aleksuss @pavel-mukhanov @vldm @slowli
+
+# Cryptocurrency examples:
+examples/cryptocurrency/* @darkeld3r @aleksuss @vitvakatu @ilammy
+examples/cryptocurrency-advanced/* @darkeld3r @aleksuss @vitvakatu @ilammy
+
+# Timestamping:
+examples/timestamping/* @darkeld3r @aleksuss @ozkriff @mvlabat @vitvakatu
+
+# Configuration service:
+services/configuration/* @darkeld3r @aleksuss @alekseysidorov @vldm @slowli
+
+# Time-oracle service:
+services/time/* @darkeld3r @aleksuss @alekseysidorov @slowli
+
+# Testkit:
+testkit/* @darkeld3r @aleksuss @alekseysidorov @slowli
+
+# Benchmarks:
+exonum/benches/* @darkeld3r @aleksuss @vldm @alekseysidorov @andreyluchin
+
+# Web api:
+exonum/src/api/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin
+
+# Encoding:
+exonum/src/encoding/* @darkeld3r @aleksuss @vldm
+
+# Events:
+exonum/src/events/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin @ilammy
```

Comment:
Pls add me to events, because I working a lot with network :)

Reply:
Done: https://github.com/exonum/exonum/pull/938